test/e2e/parallel/space_test.go

```diff
@@ -157,6 +151,9 @@ func TestSpaceRoles(t *testing.T) {
 		UntilHasLastAppliedSpaceRoles(nsTmplSet.Spec.SpaceRoles))
 	require.NoError(t, err)
 
+	ownerBinding, err := awaitilities.Host().WaitForSpaceBinding(t, ownerMUR.GetName(), ownerMUR.Name)
```

Comment:
to have it consistent :-) 
```suggestion
	ownerBinding, err := awaitilities.Host().WaitForSpaceBinding(t, ownerMUR.Name, ownerMUR.Name)
```